### PR TITLE
Fix pypi deployment by using documentation at markdown format

### PR DESCRIPTION
**Pull request by 4383**
podman README.md is markdown file who is used as a long description
so we need to specify to pypi to use markdown parser to properly
upload package

https://pypi.org/help/#description-content-type

### CODE REVIEW

**Comment by 4383** [comment]
Are you sure you fix the issue with #35 ?

The pypi documentation tell us to use `long_description_content_type="text/markdown"`

- https://pypi.org/help/#description-content-type
- https://packaging.python.org/tutorials/packaging-projects/#creating-setup-py

**Comment by 4383** [comment]
Feel free to close this PR if it's already good for you

**Comment by rhatdan** [comment]
LGTM
@jwhonce PTAL

**Comment by 4383** [comment]
FYI, take care, it's a sort of rollback on #35, but normally it's the right way to use README markdown format

**Comment by 4383** [comment]
we can test it by pushing a `0.0.3` and in a second time we need to update the README to explain how to install from pypi, I will submit a patch related to this, don't merge it until the deployment fail is not fixed.

**Comment by 4383** [comment]
openstack have the same issue

http://lists.openstack.org/pipermail/openstack-discuss/2019-July/007459.html, I don't think that this is a podman issue...

Anyway these changes can be merged since they fix some nits and bad config names.

**Comment by 4383** [comment]
Well... I think the issue here is related to:
- https://github.com/pypa/warehouse/pull/5835
- https://github.com/pypa/warehouse/issues/5890

**Comment by 4383** [comment]
According to [this comment](https://github.com/pypa/warehouse/issues/5890#issuecomment-494868157) podman seem ok:
```shell
twine check dist/*                                                                                                                      
Checking distribution dist/podman-0.0.3.dev1-py2.py3-none-any.whl: Passed
Checking distribution dist/podman-0.0.3.dev1.tar.gz: Passed
```

**Comment by 4383** [comment]
I don't know why but on `travis` the [same tests fails](https://travis-ci.org/containers/python-podman/jobs/552761880)

**Comment by 4383** [comment]
Locally I use:
```shell
$ pip list
Package           Version  
----------------- --------- 
pbr               5.3.1    
setuptools        41.0.1   
twine             1.13.0   
wheel             0.33.4   
```
and on travis:
- setuptools 41.0.1
- twine 1.13.0
- wheel 0.33.4

The used packages are the sames and the check result is different...

**Comment by 4383** [comment]
#32 we already publish sdist and bdist_wheel but the deployment is stuck by a [pypi recent update](https://github.com/pypa/warehouse/issues/5890) who reintroduce some deactivated tests on the `long_description` format...

File: setup.cfg [hunk 1]
```diff
@@ -20,6 +23,9 @@ keywords =
     varlink
     libpod
     podman
```
Comment (di) [inline setup.cfg]: I think these keywords are producing an invalid PKG-INFO file, they look like this on Travis:

```
Keywords: varlink
libpod
podman
```

But should look like:

```
Keywords: varlink,libpod,podman
```

However I'm not sure where the bug lies.

Comment (4383) [reply setup.cfg]: > I think these keywords are producing an invalid PKG-INFO file, they look like this on Travis:
> 
> ```
> Keywords: varlink
> libpod
> podman
> ```
> 
> But should look like:
> 
> ```
> Keywords: varlink,libpod,podman
> ```
> 
> However I'm not sure where the bug lies.

@di Oh thanks, good catch!

[I have already fixed this error on `pbr`](https://github.com/openstack/pbr/commit/2e3aa0d8519a35a6935aadb2e3322c58f9367fa1) but I need to verify why it's occur here.

I'll test with the inline syntaxe `keywords: varlink, libpod, podman` but I don't think it's an issue who can explain why `twine check dist/*` fail on travis.

**Comment by 4383** [comment]
Well it look like the issue is now fixed since I use the same commands that use pypa on their side.

I think we can be confident with my proposed changes, and I think we can move forward by merging these changes.

I want to keep the tests part (`twine check dist/*`) to ensure us that if we change things on `README.md` then we will detect potential markdown errors via the CI.

@di a greate thanks for all your help :)

In a second I will inspect why `pbr` doesn't manage keywords in weird way even with my previous fix.

**Comment by 4383** [comment]
@jwhonce @rhatdan PTAL

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**